Formula/mdxmini.rb

```diff
@@ -16,17 +16,15 @@ class Mdxmini < Formula
     sha256 cellar: :any, el_capitan:    "d20b94107c25833096401be6336544f283e6956758d4238e207e6a4e34fa5fdf"
   end
 
-  depends_on "sdl"
+  depends_on "sdl2"
 
   resource "test_song" do
     url "https://ftp.modland.com/pub/modules/MDX/-%20unknown/Popful%20Mail/pop-00.mdx"
     sha256 "86f21fbbaf93eb60e79fa07c759b906a782afe4e1db5c7e77a1640e6bf63fd14"
   end
 
   def install
-    # Specify Homebrew's cc
-    inreplace "mak/general.mak", "gcc", ENV.cc
-    system "make"
+    system "make", "CC=#{ENV.cc}", "LD=#{ENV.cc}"
 
     # Makefile doesn't build a dylib
     system ENV.cc, "-dynamiclib", "-install_name", "#{lib}/libmdxmini.dylib",
```

Comment:
Can we contribute this upstream?

Reply:
Sent a patch that should make it easier to build a shared library. https://github.com/mistydemeo/mdxmini/pull/3